### ISSUE TITLE
Site currently lacks accessibility features: research and implement.

### ISSUE THREAD

**Issue by sunillshastry** [sunillshastry/sunilshastry.com#18]
- Research techniques and potential fixes to improve website accessibility.
- Structure the `index.html` document accordingly as per the accessibility guidelines.
